internal/credential/vault/testing.go

```diff
@@ -634,6 +634,19 @@ func (v *TestVaultServer) LookupToken(t *testing.T, token string) *vault.Secret
 	return secret
 }
 
+// LookupTokenError calls /auth/token/lookup on v for the token. It expects the lookup to fail,
+// and returns the error received.  See
+// https://www.vaultproject.io/api-docs/auth/token#lookup-a-token.
+func (v *TestVaultServer) LookupTokenError(t *testing.T, token string) error {
```

Comment:
Why return the error when this is asserting there is an error?

Reply:
My thought was that the caller could assert it is a specific error that test is looking for.